geode-redis/src/main/java/org/apache/geode/redis/internal/data/CommandHelper.java

```diff
@@ -137,38 +137,46 @@ private RedisString checkStringType(RedisData redisData, boolean ignoreTypeMisma
     return (RedisString) redisData;
   }
 
-  RedisString getRedisString(ByteArrayWrapper key) {
+  RedisString getRedisString(ByteArrayWrapper key, boolean affectsRedisStatistics) {
     RedisData redisData = getRedisData(key, NULL_REDIS_STRING);
+
     if (redisData == NULL_REDIS_STRING) {
-      redisStats.incKeyspaceMisses();
+      if (affectsRedisStatistics)
```

Comment:
Did you run spotless before submitting this PR? Everyone else(!) ha this set up as a pre-commit hook! It's one of our standard practices. Please add a block for these single statements

Reply:
I did run spotless. This code is not incorrect and I've seen this being used elsewhere in our codebase. Why are you always nitpicking my code?

Reply:
Just because spotless didn't pick it up doesn't mean it's not part of our coding style

Reply:
> Why are you always nitpicking my code?

Reply:
If you had just made the change, this could have been long merged!